ios/Gekidou/Sources/Gekidou/PushNotification/PushNotification.swift

```diff
@@ -73,16 +73,19 @@ public class PushNotification: NSObject {
             let url = Network.default.buildApiUrl(ackNotification.serverUrl, endpoint)
             Network.default.request(
                 url, withMethod: "POST", withBody: jsonData,
-                andHeaders: headers, forServerUrl: ackNotification.serverUrl) { data, response, error in
-                    if (error != nil && ackNotification.isIdLoaded) {
-                        let backoffInSeconds = self.fibonacciBackoffsInSeconds[self.retryIndex]
```

Comment:
Doesn't this get check on line 63? Or this is somehow delayed and the index may have changed?

Reply:
It does, but it looks like with all the async code, this can reach and out of range. This was reported by sentry just once, but I thought it doesnt hurt to be safe